### DISCUSSION TITLE
no certificate?

### DISCUSSION THREAD

**Discussion by wofo99** [fedorovvl/tso_client#371] [General]
Two of my guild member see the following on startup of the client:
![image](https://github.com/fedorovvl/tso_client/assets/24462586/cfcbddf7-cdaf-44ac-8d01-888704ad24fb)

How can this be fixed?

**Comment by Cdrwbw** [comment]
Is it to play on the test server?
We've had to validate certificates 100 to 150 times and this for several months now, and so far there's no solution in sight.

**Reply by wofo99** [reply]
no, just regular login to the game
